### PR TITLE
[ie/eplus] Check if it's a pass ticket url; replace a dead url in test

### PR DESCRIPTION
**Pull request by pzhlkj6612**
**IMPORTANT**: PRs without the template will be CLOSED

### Summary

Related to #9515.

An "ex/player" url without "show_id" points to either a player page or a collection of multiple player page urls.

### Question

Maybe we should change the regex to `https?://live\.eplus\.jp/ex/player\?ib=.+` ?

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

**Comment by pzhlkj6612** [comment]
For 113758dea687d057cdfe45fb88f5fde79d687a5b , here are the test urls:

- Pass ticket index: https://live.eplus.jp/ex/player?ib=6QSsQdyRAwOFZrEHWlhRm7vocgV%2FO0YzBZ%2BaBEBg1XR%2FmbLn0R%2F048dUoAY038%2F%2F92MJ73BsoAtvUpbV6RLtDQ%3D%3D
- Player page: https://live.eplus.jp/ex/player?ib=6QSsQdyRAwOFZrEHWlhRm7vocgV%2FO0YzBZ%2BaBEBg1XR%2FmbLn0R%2F048dUoAY038%2F%2F92MJ73BsoAtvUpbV6RLtDQ%3D%3D&show_id=2371511

Comparison:

```diff
- Pass ticket index
+ Player page

  data_json = {
    "delivery_status": "...",
  
-   "archive_mode": null,
-   "is_pass_ticket": "YES",
+   "archive_mode": "ON",
+   "is_pass_ticket": "NO",
  
    "app_id": "...",
    "app_name": "...",
    "drm_mode": null
  }

```

.

**Comment by pzhlkj6612** [comment]
While there is some metadata in the pass ticket page, I don't think it makes much sense to extract it since there must not be any playable media. Any opinion?

**Comment by bashonly** [comment]
> I don't think it makes much sense to extract it since there must not be any playable media

Agreed, I wouldn't worry about it

File: yt_dlp/extractor/eplus.py [hunk 1]
```diff
@@ -16,13 +16,31 @@ class EplusIbIE(InfoExtractor):
...
+            'release_timestamp': 1608544800,
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': [
+            'Could not find the playlist URL. This event may not be accessible',
+            'No video formats found!',
+            'Requested format is not available',
```
Comment (bashonly) [inline yt_dlp/extractor/eplus.py:34]: `expected_warnings` are checked w/ regex, so we can trim these down a bit
```suggestion
            'This event may not be accessible',
            'No video formats found',
            'Requested format is not available',
```

File: yt_dlp/extractor/eplus.py [hunk 2]
```diff
@@ -124,6 +142,10 @@ def _real_extract(self, url):
         if data_json.get('drm_mode') == 'ON':
             self.report_drm(video_id)
 
+        if data_json.get('is_pass_ticket') == 'YES':
+            raise ExtractorError(
+                'Pass ticket url is not supported, please use a url points to the player page', expected=True)
```
Comment (bashonly) [inline yt_dlp/extractor/eplus.py:147]:
```suggestion
                'This URL is for a pass ticket instead of a player page', expected=True)
```

**Review by bashonly — APPROVED** [review]

**Pull request merged**